lib/cli.js

```diff
@@ -111,6 +112,19 @@ ${commandHelp()}
       cli.entries = entries;
     }
 
-    woof(cli);
+    woof(cli).catch((e) => {
+      let error = e;
+      // eslint-disable-next-line no-shadow
+      const { error: log } = console;
+
+      if (!(error instanceof Error)) {
+        error = new Error(
+          `webpack-command failed with a value of: ${inspect(error)}`
+        );
+      }
+
+      log(error.stack);
+      process.exit(1);
```

Comment:
`process.exitCode = 1` would be better

Reply:
why is that? do you have documentation that states one method is better versus another? we want to force a process exit here.

Reply:
From the Node.js docs: https://nodejs.org/api/process.html#process_process_exit_code

>Calling process.exit() will force the process to exit as quickly as possible even if there are still asynchronous operations pending that have not yet completed fully, including I/O operations to process.stdout and process.stderr.

>In most situations, it is not actually necessary to call process.exit() explicitly. The Node.js process will exit on its own if there is no additional work pending in the event loop. The process.exitCode property can be set to tell the process which exit code to use when the process exits gracefully.

Reply:
aye. so with the intent here, keeping it as-is is what we want. a full-stop to operations.

Reply:
Well, letting node flush all pending low-level IO operations is always a good idea. It's up to you to decide.

Reply:
@ooflorent gave some thought and I agree with you. Will be pushing a change shortly.